main.go

```diff
@@ -69,7 +68,16 @@ func reportSubcommand(project Project, creds GithubCredentials, repo string, pre
 	}
 
 	for _, todo := range todosToReport {
-		reportedTodo, err := todo.ReportTodo(creds, repo, prependBody+"\n"+strings.Join(todo.Body, "\n"))
+		var newline string = ""
+		switch creds.(type) {
+		case GitlabCredentials:
+			newline = "\n\n"
+		case GithubCredentials:
+			newline = "\n"
+		}
```

Comment:
I'm actually ok with double newline for GitHub. Wanted to add that for awhile. No need for that type dispatching.

Reply:
removed